src/tree-node/index.js

```diff
@@ -71,14 +65,16 @@ TreeNode.propTypes = {
     checked: PropTypes.bool,
     expanded: PropTypes.bool,
     disabled: PropTypes.bool,
-    dataset: PropTypes.object
+    dataset: PropTypes.object,
+    partiallySelected: PropTypes.bool
```

Comment:
`partiallySelected` is not used, is it really necessary here?

Reply:
You forgot to delete this line. It is no longer necessary in my opinion.

Reply:
@mrchief You forgot to delete this line. It is no longer necessary in my opinion.